Control coverage/Feature/VirtualMachine.md

```diff
@@ -1456,4 +1457,61 @@ subscription level:
 <br />
 
 ___
+## Azure_VirtualMachine_AuthN_Enable_AAD_Auth_Linux_Trial 
+
+### Display Name 
+[Trial] AAD extension must be deployed to the Linux VM
+
+### Rationale 
+Installing AAD extension on VM allows you to login into VM using Azure AD, making it possible to login user without password and improves authentication security.
+
+### Control Settings 
+```json 
+{
+     "Linux": [
```

Comment:
This looks like an invalid JSON, an array shouldn't have keys

Reply:
Fixed